src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
           jcr:primaryType="cq:Component"
-          jcr:title="AEM Forms Container"
+          jcr:title="Adaptive Form - Embed"
           sling:resourceSuperType="core/fd/components/aemform/v1/aemform"
```

Comment:
@barshat7 we should use latest version(v2) for aemform in archetype. 
cc: @rismehta